extensions/proposals/KHR_parallel_shader_compile/extension.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- vi:set sw=2 ts=4: -->
+<?xml-stylesheet href="../../extension.xsl" type="text/xsl"?>
+<proposal href="proposals/KHR_parallel_shader_compile/">
```

Comment:
I think the name should be WEBGL_parallel_shader_compile, instead of KHR_parallel_shader_compile.

Reply:
KHR_parallel_shader_compile is preferred. The WebGL working group prefers to only rename extensions into the WEBGL_ namespace when there is a significant functionality difference compared to the underlying extension. In this case, the extension is being exposed exactly as specified.